developer-docs-site/docs/tutorials/validator-node/using-aws.md

```diff
@@ -144,11 +144,11 @@ This will download all the terraform dependencies for you, in the `.terraform` f
     ---
     root_key: "0x5243ca72b0766d9e9cbf2debf6153443b01a1e0e6d086c7ea206eaf6f8043956"
     users:
-      - <username you specified in step 11>
+      - <validator_name you created in step 4>
```

Comment:
indeed, but then you need t update step 11 too

Reply:
I looked at it again, and now I think we can park these `username` vs `validator_name` changes until the next AIT round. By which time I expect we will have better scripts and node setup flow. I will revert these specific changes.